src/bikeracks/__init__.py

```diff
@@ -75,8 +78,33 @@ def submitFeedback():
                 # writing the data rows 
                 csvwriter.writerow(row)
         except Exception as e:
-            return (e, 500)
+            return (str(e), 500)
         return ('OK', 200)
+
+    # Function for processing "suggest removal" request
+    @app.route('/submitRemovalSuggestion', methods=('POST',))
+    def submitSuggestion():
+        rack_id = request.form.get("rack_id", type=float)
+        removal_reason = request.form.get("removal_reason", type=str)
+        user_id = request.form.get("user_id", type=str)
+
+        # connect to database
+        db = get_db()
+
+        try:
+        #rack_id, user_id, removal_reason, time_stamp
+            query = """
+                INSERT INTO 
+                    suggested_removals
+                        (rack_id, user_id, removal_reason, time_stamp)
+                    VALUES 
+                        (?, ?, ?, datetime('now'))"""
+            db.execute(query, (rack_id, user_id, removal_reason))
+            db.commit()
+        except Exception as e:
+            return(str(e), 500)
```

Comment:
Why return database exceptions to the user?

If you want to log, go ahead and log, but I would just let flask handle the exception in the normal way.